tests/unit/Utopia/Database/Validator/Query/OffsetTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Tests\Unit\Utopia\Database\Validator\Query;
+
+use Appwrite\Utopia\Database\Validator\Query\Base;
+use Appwrite\Utopia\Database\Validator\Query\Offset;
+use Utopia\Database\Query;
+use PHPUnit\Framework\TestCase;
+
+class OffsetTest extends TestCase
+{
+    /**
+     * @var Base
+     */
+    protected $validator = null;
+
+    public function setUp(): void
+    {
+        $this->validator = new Offset();
```

Comment:
Lets initialise the `Offset()` validator constructor with the offset explicitly for better readability. 

`new Offset(5000);`